### PR TITLE
Add nci env dea-module test scripts

### PR DESCRIPTION
**Pull request by santoshamohan**
Added new shell scripts and dependency files for testing dea module.

### CODE REVIEW

File: scripts/nci_env_deatest/datacube_config.conf [hunk 1]
```diff
@@ -0,0 +1,12 @@
+[datacube]
+db_hostname: agdcdev-db.nci.org.au
+db_port: 6432
+db_database: ncitest_datacube_db
+db_username: sm9911
```
Comment (omad) [inline scripts/nci_env_deatest/datacube_config.conf]: We probably don't need to specify the username here, to make it more easily run-able by others.

File: scripts/nci_env_deatest/Test_DEA_Module.sh [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+sh "$HOMEDIR"/dea_testscripts/index_and_ingest.sh "$MODULE" "$CONFIGFILE" "$HOMEDIR" > "$HOMEDIR"/output_files/idx_ingest/Index_Ingest_NCI_PC.log
+
+echo
+echo "==================================================================="
+echo "|  Test the following on Raijin system using PBS:                 |"
+echo "|  1) Indexing  LS7_ETM_NBAR/NBART, LS8_OLITIRS_NBAR/NBART Scenes |"
+echo "|  2) And then ingesting the products to test database            |"
+echo "==================================================================="
+echo
+qsub -v MUT="$MODULE",CONFIG_FILE="$CONFIGFILE",FL_PATH="$HOMEDIR",DC_CONF="$DCCONF" "$HOMEDIR"/dea_testscripts/pbs_script.sh
```
Comment (omad) [inline scripts/nci_env_deatest/Test_DEA_Module.sh]: What does `FL_PATH` stand for?

**Review by omad — CHANGES_REQUESTED** [review]
This looks really really good, thank you Santosh!

Just a few things before we merge it:
- It would be great to run the tests as different users. It looks to be almost possible, but still has your username/paths hard coded in a couple of places.
- I think it would be better to not duplicate the FC/Ingestion/Stats configuration files. Instead, i It should be possible to copy them from the module under test, and run a `sed -e -e 's/path/newpath/g` replacement over them. I'd expect this to be lower maintenance in the future and a slightly more thorough test in case the configurations are ever broken.
- Can you please include a README with instructions for how to run this. I'm not sure whether the intention is to copy this directory somewhere and run it, or... something else.

Cheers! 👍

File: scripts/nci_env_deatest/datacube_config.conf (hunk 1 above)
Comment (santoshamohan) [reply scripts/nci_env_deatest/datacube_config.conf]: Thanks for your feedback, Damien. 
I will make changes to the scripts as mentioned, to make it more generic and useable.
Also, I will add a README.md file and remove the username from the config file.

File: scripts/nci_env_deatest/Test_DEA_Module.sh (hunk 2 above)
Comment (santoshamohan) [reply scripts/nci_env_deatest/Test_DEA_Module.sh]: FL_PATH means File Path. Do you want me to change this to FILE_PATH?

Comment (omad) [reply scripts/nci_env_deatest/Test_DEA_Module.sh]: I'm not sure `FILE_PATH` is meaningful either. Maybe something like `TEST_BASE` or `TEST_PATH`? Depends on how and what it's used for.

Comment (santoshamohan) [reply scripts/nci_env_deatest/Test_DEA_Module.sh]: TEST_BASE would be more appropriate. I will update FL_PATH to TEST_BASE.

**Comment by santoshamohan** [comment]
Hi Damien,

I have update the comments and pushed updated code to the repo.

**Review by omad — APPROVED** [review]
Thanks

**Review by omad — APPROVED** [review]

**Pull request merged**